google_guest_agent/network/manager/netplan_linux.go

```diff
@@ -88,11 +79,22 @@ type netplanEthernet struct {
 	// Match is the interface's matching rule.
 	Match netplanMatch `yaml:"match"`
 
+	// MTU defines the interface's MTU configuration.
+	MTU *int
+
 	// DHCPv4 determines if DHCPv4 support must be enabled to such an interface.
 	DHCPv4 *bool `yaml:"dhcp4,omitempty"`
 
+	DHCP4Overrides *netplanDHCPOverrides `yaml:"dhcp4-overrides,omitempty"`
+
 	// DHCPv6 determines if DHCPv6 support must be enabled to such an interface.
 	DHCPv6 *bool `yaml:"dhcp6,omitempty"`
+
+	DHCP6Overrides *netplanDHCPOverrides `yaml:"dhcp6-overrides,omitempty"`
+}
+
+type netplanDHCPOverrides struct {
+	UseDomains bool `yaml:"use-domains,omitempty"`
```

Comment:
Can you add a comment about using this to add a search domain for whatever DNS domain the instance is in

Reply:
done. thanks.